src/openrct2-ui/Ui.cpp

```diff
@@ -38,6 +38,7 @@ int NormalisedMain(int argc, const char** argv)
 int main(int argc, const char** argv)
 #endif
 {
+    std::unique_ptr<IContext> context;
     int runGame = cmdline_run(argv, argc);
```

Comment:
The result in runGame can be 
```
    EXITCODE_FAIL = -1,
    EXITCODE_OK = 0,
    EXITCODE_CONTINUE = 1,
```
Only return EXIT_FAILURE when cmdline_run returns EXITCODE_FAIL